nf_topology/v1alpha1/nf_topology_types.go

```diff
@@ -41,13 +32,30 @@ type NFInterface struct {
 	NetworkInstanceName string `json:"networkInstanceName" yaml:"networkInstanceName"`
 }
 
+// PackageRevisionReference is a temporary replica of PackageRevisionReference used for the
+// ONE Summit
+type PackageRevisionReference struct {
+	// Namespace is the namespace for both the repository and package revision
+	// +optional
+	Namespace string `json:"namespace,omitempty"`
+
+	// Repository is the name of the repository containing the package
+	RepositoryName string `json:"repository"`
+
+	// PackageName is the name of the package for the revision
+	PackageName string `json:"packageName"`
+
+	// Revision is the specific version number of the revision of the package
+	Revision string `json:"revision"`
+}
+
 // NFTemplate defines the template for deployment of an instance of a NF
 type NFTemplate struct {
 	// NFType specifies the type of NF this template is specifying
-	NFType NFType `json:"nfType" yaml:"nfType"`
+	NFType string `json:"nfType" yaml:"nfType"`
```

Comment:
Yes, better not to use an enum to avoid having to identify all possible types of NFs.